src/xcode/ENA/ENA/Source/Models/Task Scheduling/ENATaskScheduler.swift

```diff
@@ -22,14 +22,17 @@ import UIKit
 enum ENATaskIdentifier: String, CaseIterable {
 	// only one task identifier is allowed have the .exposure-notification suffix
 	case primaryBackgroundTask = "exposure-notification"
+	case secondaryBackgroundTask = "fetch-test-results"
 
 	var backgroundTaskScheduleInterval: TimeInterval? {
 		switch self {
 		case .primaryBackgroundTask: return nil
```

Comment:
what is the reason for having a switch statement here, if the return value is `nil` regardless?

Reply:
It's using the standard enum syntax.

Reply:
Why is it not possible to change
```swift

var backgroundTaskScheduleInterval: TimeInterval? {
		switch self {
		case .exposureNotification: return nil
		case .fetchTestResults: return nil
		}
	}
```

to 

```swift

var backgroundTaskScheduleInterval: TimeInterval?
```

if it is nil anyways, regardless of the case.

Reply:
Apologies, I thought you were referring to the case lines above. I've updated the fetch-test-results to a 2hr schedule

Reply:
No problem. Thanks for fixing it! 😄